Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/Handler.purs

```diff
@@ -285,11 +285,26 @@ callSupportHandler updatedState = do
 reOpenIssueHandler :: ReportIssueChatScreenState -> FlowBT String FlowState
 reOpenIssueHandler updatedState = do
   let language = fetchLanguage $ getLanguageLocale languageKey
-  let updateIssueReqBody = UpdateIssueReqBody {status : "OPEN"}
-  (UpdateIssueRes _) <- Remote.updateIssue (fromMaybe "" updatedState.data.issueId) language updateIssueReqBody
-  pure unit
-  modifyScreenState $ ReportIssueChatScreenStateType (\_ -> updatedState {props {isResolved = false}})
-  App.BackT $ App.NoBack <$> (pure $ IssueReportChatScreenFlow)
+      selectedOptionId = fromMaybe "" $ map (\option -> option.issueOptionId) updatedState.data.selectedOption
+      updateIssueReqBody = UpdateIssueReqBody { status : "REOPENED" }
+  (GetOptionsRes _) <- Remote.getOptionsBT language updatedState.data.categoryId selectedOptionId (fromMaybe "" updatedState.data.issueId)
+  (UpdateIssueRes updateIssueRes) <- Remote.updateIssue (fromMaybe "" updatedState.data.issueId) language updateIssueReqBody
+  let messages' = DA.mapWithIndex (\index (Message currMessage) -> 
+                                      makeChatComponent' (reportIssueMessageTransformer currMessage.message) "Bot" (getCurrentUTC "") "Text" (500 * (index + 1))
+                                  ) updateIssueRes.messages
+  modifyScreenState $ ReportIssueChatScreenStateType (\_ -> 
+    updatedState { 
+      data = updatedState.data {
```

Comment:
can you wrap the data like string 
like this `"data" = ...`
it will cause the issue in code formatters.

Reply:
Done